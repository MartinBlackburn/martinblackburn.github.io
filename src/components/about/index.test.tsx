// libraries
import { render } from "@testing-library/react";

// component under test
import About from ".";

describe("About component", () => {
    test("renders correctly", () => {
        const { asFragment } = render(<About />);

        expect(asFragment()).toMatchSnapshot();
    });
});
