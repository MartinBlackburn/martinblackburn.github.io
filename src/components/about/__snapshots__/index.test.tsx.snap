// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders correctly 1`] = `
<DocumentFragment>
  <section
    class="about"
  >
    <h1>
      I'm an Engineering Manager, specialising in 
      <strong>
        JavaScript/TypeScript
      </strong>
      , 
      <strong>
        React
      </strong>
       and 
      <strong>
        mentoring other aspiring developers
      </strong>
      .
    </h1>
  </section>
</DocumentFragment>
`;
